_posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Hackathon Co-Afina 2023 - Latin American Students Drive Innovation with Open Data
+authors:
+- camachoreina
+image: https://www.software.ac.uk/sites/default/files/2023-12/unnamed.jpg
+photos:
+  name: Image by Reina Camacho Toro
+  license: CC-BY
+  url: https://github.com/camachoreina
+---
+
+_This blog post was original shared by Software Sustainability Institute, and found [here](https://www.software.ac.uk/blog/hackathon-co-afina-2023-latin-american-students-drive-innovation-open-data), as authored by R. Camacho Toro, Y. Briceño and A. Martínez._
```

Comment:
Slight rewording suggestion, feel free to ignore:
This blog post was original shared by Software Sustainability Institute, and can be found [here](https://www.software.ac.uk/blog/hackathon-co-afina-2023-latin-american-students-drive-innovation-open-data), as authored by R. Camacho Toro, Y. Briceño and A. Martínez.

Reply:
Hello! Using CC BY is perfect. Shall we added to the figure directly? Or the OLS web already has CC by default?

Reply:
Thanks a lot!

Reply:
Thanks @camachoreina - OLS website content is CC-BY-SA unless specified differently, so this is perfect!